metpetdb_api/api/chemical_analyses/lib/query.py

```diff
@@ -8,6 +8,14 @@ def chemical_analysis_query(user, params, qs):
     else:
         qs = qs.filter(Q(owner=user) | Q(public_data=True))
 
+    if params.get('provenance'):
+        if(params.get('provenance')=="Public"):
+            qs = qs.filter(public_data=True)
+        elif(params.get('provenance') == "Private"):
+            qs = qs.filter(public_data=False | Q(owner=user))
+        else:
```

Comment:
Same as https://github.com/metpetdb/api_v2/pull/7/files#r111210630. This can be removed.